sale_timesheet_purchase/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2019 Odoo S.A.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Sales Timesheet Purchase",
+    "category": "Hidden",
+    "version": "12.0.1.0.0",
+    "author": "Odoo S.A., Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "summary": "Bridge module between Sales Timesheet and Purchase",
+    "depends": ["sale_timesheet", "purchase"],
+    "installable": True,
+    "auto_install": False,
```

Comment:
A bridge module is better auto installed
```suggestion
    "auto_install": True,
```

Reply:
Mn this is a problem, with just be present in other environment this module stick as a glue... bad bad Idea :-/